src/main/java/org/sunbird/searchby/service/SearchByService.java

```diff
@@ -408,4 +412,159 @@ private List<FracCommonInfo> getMasterPositionList(List<String> positionNameList
 		}
 		return positionList;
 	}
+
+	public Collection<CompetencyInfo> listCompetenciesByOrg(String orgId, String userToken) {
+		 //  id's of content (contentSearch for every ids and implement facets)
+		try {
+			List<String> competenciesList = redisCacheMgr.hget(Constants.TOP_COMMENTS_ORG_REDIS_KEY, cbExtServerProperties.getRedisInsightIndex(), orgId);
+			Map<String, CompetencyInfo> competencyMap = new HashMap<>();
+			if (CollectionUtils.isEmpty(competencyMap)) {
+				logger.info("Initializing/Refreshing the Cache Value for Key : " + Constants.COMPETENCY_CACHE_NAME);
+//				competencyMap = listCompetencyDetails(userToken, competenciesList);
```

Comment:
suppose to be uncommented ?

Reply:
we have planned to move this API in another PR. so removed the code.